### PR TITLE
First version of Loopback tutorial

### LINKED ISSUE #523: Add tutorial from LoopBack
Request to include the existing tutorial from the LoopBack site into the Appsody site.

Source is [here](https://github.com/strongloop/loopback-next/blob/master/docs/site/Appsody-LoopBack.md)

Conversion/editing can start ahead of https://github.com/appsody/website/issues/195

"take the ‘Scaffolding a LoopBack 4 Application using Appsody’ and ‘Building Your Application’ sections from that tutorial, and add as a scenario for ‘creating a todo application with loopback 4’ in the docs."

### PR DESCRIPTION
**Pull request by helenmasters**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
First version of the Loopback tutorial. It will also need reviewing by a member of the Loopback team.

## Related Issues
Fixes #523

### CODE REVIEW

File: content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
+---
+title: 'Developing and Deploying LoopBack Applications with Appsody'
+date: ""
```
Comment (SueChaplain) [inline content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md:3]: Date and author needed from the Loopback team.

**Comment by KyeMaloy97** [comment]
Checked it and everything seems fine to me, from a formatting end point. Needs a name and a date in the frontmatter though before it is merged.

File: content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
+---
+title: "Developing and Deploying LoopBack Applications with Appsody"
+date: ""
+author: ""
```
Comment (dhmlau) [inline content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md:4]:
```suggestion
author: "Dominique Emond"

```

File: content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md (hunk 1 above)
Comment (dhmlau) [reply content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md:3]: We don't have preference on the publish date. @SueChaplain, perhaps you can suggest one that fits your publish schedule? Thanks.

**Comment by KyeMaloy97** [comment]
@dhmlau the date is just for when the post would originally have been made, or when the tutorial was written. I had a look and the original date this was made was the 19th Dec 2019 so I will just add that as the date.

File: content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+
+Appsody helps developers build containerized applications for the cloud that are ready to be deployed to Kubernetes without the user needing to be an expert on the underlying container technology.
+
+The LoopBack team has recently contributed the [Node.js LoopBack stack](https://github.com/appsody/stacks/tree/master/incubator/nodejs-loopback) which extends the [Node.js stack](https://github.com/appsody/stacks/tree/master/incubator/nodejs) and provides a powerful solution to build open APIs and Microservices in TypeScript with [LoopBack](https://loopback.io), an open source Node.js API framework. It is based on [LoopBack 4](https://github.com/strongloop/loopback-next).
+
+This tutorial is going to show you how to scaffold, run, stop, debug, and test a LoopBack 4 application in the Appsody development environment.
+
+## Prerequisites
+
+In order to follow this tutorial, you need to install the Appsody CLI, which also requires you to have Docker installed, along with a Node.js application to cloud package.
```
Comment (emonddr) [inline content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md:19]: > along with a Node.js application to cloud package.

Not sure what this means. What cloud, what package?

File: content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md [hunk 4]
```diff
@@ -0,0 +1,342 @@
...
+
+You are going to use the `todo list` application from the
+[Todo Tutorial](https://loopback.io/doc/en/lb4/todo-tutorial.html) as an
+example.
+
+### Add the Todo model
+
+To add the Todo model you need the `'lb4'` application generator installed.
+
+#### Install the 'lb4' application generator
```
Comment (emonddr) [inline content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md:154]: > Install the 'lb4' application generator

Install the LoopBack 4 CLI

File: content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md [hunk 5]
```diff
@@ -0,0 +1,342 @@
...
+
+[Install the 'lb4' application generator](https://loopback.io/doc/en/lb4/Getting-started.html) by entering the following command:
```
Comment (emonddr) [inline content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md:156]: > Install the 'lb4' application generator

Install the LoopBack 4 command line interface

File: content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md (hunk 3 above)
Comment (emonddr) [reply content/tutorials/DevelopAndDeployLoopbackAppsWithAppsody.md:19]: Actually, now that I've read the entire article, you don't actually deploy to the cloud, so there's no no need for 
`along with a Node.js application to cloud package.`. :)

**Comment by emonddr** [comment]
Thank you for addressing my comments. Looks great :)

**Pull request merged**